src/diffusion.jl

```diff
@@ -96,13 +97,13 @@ The variables for diffusion problem:
 $(FIELDS)
 """
 struct Vars{Aphys, Atrans} <: AbstractVars
-    "tracer concentration"
+    "tracer concentration ``c``"
     c :: Aphys
-    "tracer concentration derivative ∂ₓc"
+    "tracer concentration derivative ``∂ₓc``"
    cx :: Aphys
-    "Fourier transform of tracer concentration"
+    "Fourier transform of tracer concentration ``c``"
```

Comment:
$\hat{c}$?

Reply:
I removed the symbols to avoid ambiguity, e.g. $\hat{c}$ is not exactly `ch`. They are the same up to a normalization factor. There is a discussion in [docs/grids](https://fourierflows.github.io/FourierFlowsDocumentation/dev/grids/) about this.